sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -363,44 +363,121 @@ const alwaysBeDeletedRule = "alwaysbedeletedrule";
         should.equal(response._response.status, 200);
       });
 
-      it(`Get on non-existent ${entityType} entity returns empty response`, async () => {
+      it(`Get on non-existent ${entityType} entity throws an error`, async () => {
         let response;
-
-        let skipTest = false;
         switch (entityType) {
           case EntityType.QUEUE:
+            try {
+              await deleteEntity(entityType, alwaysBeExistingQueue);
+            } catch (err) {
+              console.log("Ignoring clean up step");
+            }
+            try {
+              await getEntity(entityType, alwaysBeExistingQueue);
+            } catch (err) {
+              response = err;
```

Comment:
It is misleading that we are assigning the error to a variable named "response".
I was glancing at the code a few lines below wondering how is that we have a "response" from the get operation while we expect it throw error.

Please consider renaming the variable to something which has "error" or "err" in the name

Reply:
updated